code/modules/organ_research/machinery.dm

```diff
@@ -0,0 +1,259 @@
+/obj/machinery/ornd
+	icon = 'icons/obj/machines/organResearch.dmi'
+	density = TRUE
+	anchored = TRUE
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 50
+	active_power_usage = 300
+	var/busy = FALSE
+	var/hacked = FALSE
+	var/disabled = 0
+	var/shocked = FALSE
+	var/obj/machinery/computer/orndconsole/linked_console
+	var/datum/ornd/refDatum
+
+/obj/machinery/ornd/Initialize()
+	.=..()
+	refDatum = new /datum/ornd

```

Comment:
/datum/ornd needs to be a "global" and refdatum just pulls from that instead of creating a new one each time.